src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLIntegTestCase.java

```diff
@@ -107,23 +106,25 @@ protected boolean preserveClusterUponCompletion() {
     }
 
     @AfterClass
-    public static void dumpCoverage() throws IOException, MalformedObjectNameException {
+    public static void dumpCoverage() {
         // jacoco.dir is set in sqlplugin-coverage.gradle, if it doesn't exist we don't
         // want to collect coverage so we can return early
         String jacocoBuildPath = System.getProperty("jacoco.dir");
-        if (jacocoBuildPath.isEmpty()) {
+        if (jacocoBuildPath == null || jacocoBuildPath.isEmpty()) {
```

Comment:
np: `isNullOrEmpty` in Strings or StringUtils is cleaner.

Reply:
good catch, use Strings.isNullOrEmpty(Guava) instead.